conans/client/command.py

```diff
@@ -564,7 +564,7 @@ def config(self, *args):
 
         install_subparser.add_argument("--verify-ssl", nargs="?", default="True",
                                        help='Verify SSL connection when downloading file')
-        install_subparser.add_argument("--type", "-t", choices=["git"],
+        install_subparser.add_argument("--type", "-t", choices=["git", "dir", "file", "url"],
```

Comment:
Please provide a test that covers this case.

Reply:
Done!